test/Exporter/Test/Integration/Writer/KnpSnappyWriterTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Integration\Writer;
+
+use Exporter\Writer\KnpSnappyWriter;
+use Knp\Snappy\Pdf;
+
+/**
+ * Tests the KnpSnapyWriter.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class KnpSnappyWriterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var string relative path to the test file
+     */
+    private $filename;
+
+    /**
+     * @var string absolute path to the wkhtmltopdf binary
+     */
+    private $wkhtmltopdf;
+
+    public function setUp()
```

Comment:
For some reason, I thought opening the accessibility of a method was forbidden… turns out it is not : https://3v4l.org/4XNQB . Closing it is : https://3v4l.org/0FrGI And you are right, this method should be protected